Patch-Source: https://sources.debian.org/data/main/libh/libhtml-gumbo-perl/0.18-5/debian/patches/no-alien-libgumbo.patch
--
Description: remove usage of Alien::LibGumbo; use ExtUtils::PkgConfig in Build.PL
Origin: vendor
Forwarded: not-needed
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2018-07-20

--- a/Build.PL
+++ b/Build.PL
@@ -1,7 +1,7 @@
 use Module::Build;
-use Alien::LibGumbo;
+use ExtUtils::PkgConfig;
 
-my $alien = Alien::LibGumbo->new;
+my %pkg_info = ExtUtils::PkgConfig->find('gumbo');
 my $builder = Module::Build->new(
     module_name => 'HTML::Gumbo',
     license => 'perl',
@@ -27,7 +27,7 @@
 
     add_to_cleanup => [ 'HTML-Gumbo-*' ],
 
-    extra_compiler_flags => $alien->cflags(),
-    extra_linker_flags   => $alien->libs(),
+    extra_compiler_flags => $pkg_info{cflags},
+    extra_linker_flags   => $pkg_info{libs},
 );
 $builder->create_build_script;
--- a/lib/HTML/Gumbo.pm
+++ b/lib/HTML/Gumbo.pm
@@ -4,7 +4,6 @@
 
 package HTML::Gumbo;
 
-use Alien::LibGumbo 0.03;
 our $VERSION = '0.18';
 
 require XSLoader;
